focus-group.js

```diff
@@ -13,7 +13,7 @@ function focus(current, next) {
 }
 
 function findGroupNodeByEventTarget(target) {
-  let fg = target.closest('[focusgroup]')
+  let fg = target.closest('[focusgroup]:not([focusgroup="none"])')
```

Comment:
I am not sure that we need this check.

I can’t imagine real use case for it. Do we have an example in spec?

Reply:
Without the `:not([focusgroup="none"])` filter, an element with `focusgroup="none"` might be incorrectly treated as a valid focus group. This could create the illusion of a subgroup inside the main focus group. However, according to the specification elements with `focusgroup="none"` and their descendants should be fully excluded from navigation.

In this scenario:
```
<div focusgroup="block" id="group1">
  <div focusgroup="none" id="group2">
    <button>Button</button>
  </div>
</div>
```

https://open-ui.org/components/focusgroup.explainer/#opting-out
> _"Focusgroup definitions assigned to an element create focusgroup candidates that include the element itself and all its shadow-inclusive descendant elements. Any element within that focusgroup scope that is (or becomes) focusable will automatically become a focusgroup item belonging to its ancestor’s focusgroup."_